.github/workflows/validate-cg-manifest.sh

```diff
@@ -13,6 +13,8 @@
 
 # Ignore some specs, mostly those with Source0 files that are not from an external source, or have very odd URLs
 ignore_list=" \
+  Cython \
```

Comment:
You should remove this since you arent updating Cython

Reply:
Done